manifests/01-registry-credentials-request.yaml

```diff
@@ -31,10 +31,10 @@ spec:
       - s3:GetLifecycleConfiguration
       - s3:GetBucketLocation
       - s3:ListBucket
-      - s3:HeadBucket
```

Comment:
I only discovered this while digging into the issues I was having before I realized I needed to add the new s3:ListMultipartUploadParts permission when running with STS credentials ¯\_(ツ)_/¯

Reply:
Could you confirm that this permission removal won't affect our calls to `HeadBucketWithContext` even when using access key and secret? Just trying to be on the safe side here :-)

Reply:
s3:HeadBucket is not a real permission. Manually adding "s3:HeadBucket" to an IAM policy (because you can't select it through the GUI) and viewing the policy in the GUI visual editor shows the following message:
```
 Unrecognized actions
IAM does not recognize one or more actions. The action name might include a typo or might be part of a previewed or custom service.
s3:HeadBucket
```